### ISSUE TITLE
krita redirect causes download to fail

### ISSUE THREAD

**Issue by BenLubar** [lukesampson/scoop-extras#2728]
The file: https://download.kde.org/stable/krita/4.2.5/krita-x64-4.2.5.zip

The request:
```
GET /stable/krita/4.2.5/krita-x64-4.2.5.zip HTTP/1.1
User-Agent: Scoop/1.0 (+http://scoop.sh/) PowerShell/6.2 (Windows NT 10.0; Win64; x64; Core)
Referer: https://download.kde.org/stable/krita/4.2.5/
Connection: Keep-Alive
Host: download.kde.org
```

The response:
```
HTTP/1.1 302 Found
Date: Fri, 23 Aug 2019 18:19:32 GMT
Server: Apache/2.4.29 (Ubuntu)
X-MirrorBrain-Mirror: ftp.ussg.iu.edu
X-MirrorBrain-Realm: other
Link: <https://download.kde.org/stable/krita/4.2.5/krita-x64-4.2.5.zip.meta4>; rel=describedby; type="application/metalink4+xml"
Link: <http://ftp.ussg.iu.edu/kde/stable/krita/4.2.5/krita-x64-4.2.5.zip>; rel=duplicate; pri=1; geo=us
Link: <http://mirror.retentionrange.co.bw/kde/stable/krita/4.2.5/krita-x64-4.2.5.zip>; rel=duplicate; pri=2; geo=bw
Link: <http://ftp.pbone.net/pub/kde/stable/krita/4.2.5/krita-x64-4.2.5.zip>; rel=duplicate; pri=3; geo=pl
Link: <http://mirror.kumi.systems/kde/ftp/stable/krita/4.2.5/krita-x64-4.2.5.zip>; rel=duplicate; pri=4; geo=de
Link: <http://ftp.is.co.za/mirror/ftp.kde.org/stable/krita/4.2.5/krita-x64-4.2.5.zip>; rel=duplicate; pri=5; geo=za
Digest: MD5=3WY3eYctCLgAdjZKk6CosQ==
Digest: SHA=V9TlByrXRxj41DM4FuwcH8Hr3i8=
Digest: SHA-256=HKwUnkpb0BIKVkMiaaM22aweR1zM4qdJFCFB2G84gKU=
Location: http://ftp.ussg.iu.edu/kde/stable/krita/4.2.5/krita-x64-4.2.5.zip
Cache-Control: max-age=0
Expires: Fri, 23 Aug 2019 18:19:32 GMT
Content-Length: 332
Keep-Alive: timeout=5, max=100
Connection: Keep-Alive
Content-Type: text/html; charset=iso-8859-1

<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>302 Found</title>
</head><body>
<h1>Found</h1>
<p>The document has moved <a href="http://ftp.ussg.iu.edu/kde/stable/krita/4.2.5/krita-x64-4.2.5.zip">here</a>.</p>
<hr>
<address>Apache/2.4.29 (Ubuntu) Server at download.kde.org Port 443</address>
</body></html>
```

It appears that *all* of the download links on https://download.kde.org/stable/krita/4.2.5/krita-x64-4.2.5.zip.mirrorlist are non-HTTPS.

**Comment by r15ch13** [comment]
Some of them actually support HTTPS but are not listed as such. 
e.g. 
https://ftp.gwdg.de/pub/linux/kde/stable/krita/4.2.5/
https://mirror.hs-esslingen.de/Mirrors/ftp.kde.org/pub/kde/stable/krita/4.2.5/

**Comment by Ash258** [comment]
Fixed with e6b355eae093df70db85a6fc7850f6568bb6e7d9
